pkg/controller/config-controller.go

```diff
@@ -0,0 +1,396 @@
+package controller
+
+import (
+	"time"
+
+	"github.com/golang/glog"
+	routev1 "github.com/openshift/api/route/v1"
+	routeinformers "github.com/openshift/client-go/route/informers/externalversions/route/v1"
+	routelisters "github.com/openshift/client-go/route/listers/route/v1"
+	"github.com/pkg/errors"
+	extensionsv1beta1 "k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	extensioninformers "k8s.io/client-go/informers/extensions/v1beta1"
+	"k8s.io/client-go/kubernetes"
+	extensionlisters "k8s.io/client-go/listers/extensions/v1beta1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
+	cdiclientset "kubevirt.io/containerized-data-importer/pkg/client/clientset/versioned"
+	clientset "kubevirt.io/containerized-data-importer/pkg/client/clientset/versioned"
```

Comment:
"kubevirt.io/containerized-data-importer/pkg/client/clientset/versioned" imported twice.

Reply:
Can we get rid of one of these imports?

Reply:
done - I was sure I got rid of it